packages/itensor/quri_parts/itensor/estimator.py

```diff
@@ -0,0 +1,205 @@
+    _sequential_estimate: Callable[
+        [Any, Sequence[tuple[Estimatable, ITensorStateT]]], Sequence[Estimate[complex]]
+    ],
+    _sequential_estimate_single_state: Callable[
+        [ITensorStateT, Sequence[Estimatable]], Sequence[Estimate[complex]]
+    ],
+    operators: Collection[Estimatable],
+    states: Collection[ITensorStateT],
+    executor: Optional["Executor"],
+    concurrency: int = 1,
+) -> Sequence[Estimate[complex]]:
+    num_ops = len(operators)
+    num_states = len(states)
+
+    if num_ops == 0:
+        raise ValueError("No operator specified.")
+
+    if num_states == 0:
+        raise ValueError("No state specified.")
+
+    if num_ops > 1 and num_states > 1 and num_ops != num_states:
+        raise ValueError(
+            f"Number of operators ({num_ops}) does not match"
+            f"number of states ({num_states})."
+        )
+
+    if num_states == 1:
+        return execute_concurrently(
+            _sequential_estimate_single_state,
+            next(iter(states)),
+            operators,
+            executor,
+            concurrency,
+        )
+    else:
+        if num_ops == 1:
+            operators = [next(iter(operators))] * num_states
+        return execute_concurrently(
+            _sequential_estimate, None, zip(operators, states), executor, concurrency
+        )
+
+
+# For now, this function works when the executor is defined like below
+# `with ProcessPoolExecutor(max_workers=2, mp_context=get_context("spawn"))
+# as executor:`
+def create_itensor_mps_concurrent_estimator(
+    executor: Optional["Executor"] = None, concurrency: int = 1
+) -> ConcurrentQuantumEstimator[ITensorStateT]:
+    """Returns a :class:`~ConcurrentQuantumEstimator` that uses ITensor MPS
+    simulator to calculate expectation values."""
+
+    def estimator(
+        operators: Collection[Estimatable],
+        states: Collection[ITensorStateT],
+    ) -> Iterable[Estimate[complex]]:
+        return _concurrent_estimate(
+            _sequential_estimate,
+            _sequential_estimate_single_state,
+            operators,
+            states,
+            executor,
+            concurrency,
+        )
+
+    return estimator
+
+
+def _sequential_parametric_estimate(
+    op_state: tuple[Estimatable, ITensorParametricStateT],
+    params: Sequence[Sequence[float]],
+) -> Sequence[Estimate[complex]]:
+    operator, state = op_state
+    estimates = []
+    estimator = create_parametric_estimator(create_itensor_mps_estimator())
+    for param in params:
+        estimates.append(estimator(operator, state, param))
+    return estimates
+
+
+def create_itensor_mps_parametric_estimator() -> ParametricQuantumEstimator[
+    ITensorParametricStateT
+]:
+    def estimator(
+        operator: Estimatable, state: ITensorParametricStateT, param: Sequence[float]
+    ) -> Estimate[complex]:
+        ests = _sequential_parametric_estimate((operator, state), [param])
+        return ests[0]
+
+    return estimator
```

Comment:
I guess you can write like:
```suggestion
    return create_parametric_estimator(create_itensor_mps_estimator())
```